app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,7 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
+		if [[ $(tc-getCC) == *clang* ]] ; then
```

Comment:
if you want to be credited with the fix, then leave this PR open, but as I said please also extend it to the other ebuilds. after all you don't want that bug re-introduced when the next libreoffice version bump is created out of an unfixed live ebuild.